src/components/structures/RightPanel.js

```diff
@@ -31,7 +31,6 @@ module.exports = React.createClass({
     displayName: 'RightPanel',
 
     propTypes: {
-        userId: React.PropTypes.string, // if showing an orphaned MemberInfo page, this is set
```

Comment:
Yeah, I'm assuming we'd be consistent and eventually move to using dispatches everywhere.